carto/sync_tables.py

```diff
@@ -29,7 +29,7 @@
 API_FORCE_SYNC_SUFFIX = 'sync_now'
 
 
-class SyncTableJob(AsyncResource):
+class SyncTableJob(WarnAsyncResource):
```

Comment:
No need to add the WarnAsyncResource since this is a [public documented API](https://carto.com/developers/import-api/guides/sync-tables/) 😅

I guess @javitonino concerns about returning all the sync fields in the response were in the `/viz` endpoint which is already marked as `WarnAsyncResource`.

On the other hand, I've double checked we have public documented all the attributes in `SynchronizationField` so it's OK if we expose all of them in our SDK.

Reply:
done ,waiting for the tests